src/scripts/wizarduserpage.js

```diff
@@ -45,6 +52,7 @@ define(["loading", "globalize", "dashboardcss", "emby-input", "emby-button", "em
             apiClient = getApiClient();
         apiClient.getJSON(apiClient.getUrl("Startup/User")).then(function(user) {
             page.querySelector("#txtUsername").value = user.Name || "", loading.hide()
+            page.querySelector("#txtManualPassword").value = user.Password || "", loading.hide()
```

Comment:
This will call `loading.hide() twice` and it should be deuglified.
```suggestion
            page.querySelector("#txtUsername").value = user.Name || "";
            page.querySelector("#txtManualPassword").value = user.Password || "";
            loading.hide();
```

Reply:
Fixed